docs/index.html

```diff
@@ -109,14 +109,14 @@
           font-variation-settings: "wght" 0;
         }
         to {
-          font-variation-settings: "wght" 1000;
+          font-variation-settings: "wght" 150;
```

Comment:
no joosh what have you done? this is so wrong

Reply:
this was just an experiment for my font